karam-karim/task-1/style.css

```diff
@@ -0,0 +1,100 @@
+body{
+    margin: 0;
+    padding: 0;
+    background-color: rgb(228,229,231);
+    font-family: monospace;
+    font-size: 1.5em;
+}
+
+.container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container > .header{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 20px;
+}
+
+.container > .header > h2, .container > .header > p{
+    color: rgb(83,82,82);
+    font-weight: bold;
+}
+
+.container > .header > p {
+    font-size: 0.7em;
+    width: 70%;
+}
+
+
+.container > .body{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container > .body > .langs {
+    display: flex;
+    flex-wrap: wrap;
+    width: 70%;
+}
+
+.container > .body > .langs > span {
```

Comment:
you can instead make this shorter ?

Reply:
I could add a class or id to the element to target them directly, the reason that I made it this way it's easier to read for me and understand what i'm styling, but if you don't like this type of selectors I can change it :smiley: